src/geometric/problem/MainShapes.java

```diff
@@ -8,12 +8,14 @@ public static void main(String[] args) {
         getArray(circle, triangle, rectangle);
         afisariEx1(circle, triangle, rectangle);
     }
+
     private static void getArray(Circle circle, Triangle triangle, Rectangle rectangle) {
```

Comment:
what does this method ?

Reply:
Testing

To test the classes, write code 

> **that creates an array of 3 shapes:**

● A circle of radius 3
● A equilateral triangle with the edges of length 8
● A rectangle; pick any values